pkg/apis/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Masterminds/semver v1.4.2
-	github.com/aws/aws-sdk-go v1.30.9 // indirect
+	github.com/aws/aws-sdk-go v1.44.72 // indirect
```

Comment:
Do we need `aws-sdk-go` package in the `apis` package?

Reply:
Maybe called by other package, not our code. i will try to find it.

Reply:
$ go mod why -m github.com/aws/aws-sdk-go
# github.com/aws/aws-sdk-go
github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1
github.com/prometheus/prometheus/config
github.com/aws/aws-sdk-go/aws/ec2metadata